commands.go

```diff
@@ -57,7 +57,21 @@ func Commands(metaPtr *command.Meta) map[string]cli.CommandFactory {
 				Meta: meta,
 			}, nil
 		},
-
+		"fs-list": func() (cli.Command, error) {
+			return &command.FSListCommand{
+				Meta: meta,
+			}, nil
+		},
+		"fs-stat": func() (cli.Command, error) {
+			return &command.FSStatCommand{
+				Meta: meta,
+			}, nil
+		},
+		"fs-cat": func() (cli.Command, error) {
```

Comment:
There should be one `fs` command and the rest (list, stat, cat) should be subcommands:
https://github.com/mitchellh/cli/pull/27